lib/tasks/cloudfoundry.rake

```diff
@@ -0,0 +1,7 @@
+namespace :cf do
+  desc "Only run on the first application instance"
+  task :on_first_instance do # rubocop:disable Rails/RakeEnvironment
+    instance_index = JSON.parse(ENV["VCAP_APPLICATION"])["instance_index"] rescue nil
+    exit(0) unless instance_index.zero?
```

Comment:
This isn't right, as regardless the exit code will be `0` as `rake` has successfully completed execution

```ruby
instance_index = 0
exit(0) unless instance_index.zero?

# > bundle exec rake cf:on_first_instance                                                                                                                                                   
# > echo $?                                                                                                                                                                                 
# 0
```


```ruby
instance_index = 1
exit(0) unless instance_index.zero?

# > bundle exec rake cf:on_first_instance                                                                                                                                                   
# > echo $?                                                                                                                                                                                 
# 0
```


Since `instance_index` can be rescued with `nil` it be better to use `lonely operator`, instead of being faced with 

```bash
NoMethodError: undefined method `zero?' for nil:NilClass
```

So you probably want it this way round
```suggestion
    exit(1) if instance_index&.nonzero?
```